pandas/plotting/_core.py

```diff
@@ -862,10 +881,14 @@ def line(self, x=None, y=None, **kwargs):
 
             >>> lines = df.plot.line(x='pig', y='horse')
         """
-        return self(kind="line", x=x, y=y, **kwargs)
 
-    def bar(self, x=None, y=None, **kwargs):
-        """
+    @Appender(_line_docs % _shared_docs)
+    def line(self, x=None, y=None, figsize=None, subplots=False, sharex=None,
```

Comment:
While doing this you might as well make all of the new keywords "keyword only", i.e. don't allow them to be called positionally. Easier to do now than to try and go back and do it later